test.rs

```diff
@@ -0,0 +1,32 @@
+type Array<T, const N: usize> = [T; N];
+
+pub type A0 = [i32; 3]; // printed correctly
+pub type A1 = Array<i32, 3>; // printed INCORRECTLY as `[i32; T]`
+pub struct B0(pub [i32; 3]); // printed correctly
+pub struct B1(pub Array<i32, 3>); // printed INCORRECTLY as `B1(pub [i32; T])`
```

Comment:
Did you add this file by accident? It doesn't seem to be related to this PR.

Reply:
Yes :sweat_smile: